pkg/node/node_test.go

```diff
@@ -61,7 +63,10 @@ func TestDryRun(t *testing.T) {
 	tNode, err := node.New(config.Config{DryRun: true})
 	h.Ok(t, err)
 
-	err = tNode.CordonAndDrain(nodeName, "cordonReason")
+	fakeRecorder := record.NewFakeRecorder(100)
```

Comment:
@brycahta asked https://github.com/aws/aws-node-termination-handler/pull/631/files#r868317617

> * Make this const please
> * Is it necessary for the buffer size to be this large?

A `record.FakeRecorder` object can't be constant. We could use a global var but we could incur in concurrency problems when testing, I think it's better to just instantiate a new var scoped to every unit test.

Regarding the buffer size, I lowered it to 10.

Reply:
Ah gotcha. Makes sense, just pushed a commit to create and use the `recorderBufferSize` const.